db/migrations/20171109145542_eventum_custom_filter_multiselect.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Eventum (Issue Tracking System) package.
+ *
+ * @copyright (c) Eventum Team
+ * @license GNU General Public License, version 2 or later (GPL-2+)
+ *
+ * For the full copyright and license information,
+ * please see the COPYING and AUTHORS files
+ * that were distributed with this source code.
+ */
+
+use Eventum\Db\AbstractMigration;
+
+class EventumCustomFilterMultiselect extends AbstractMigration
+{
+    const COLUMNS = [
+        'cst_iss_pri_id' => 'cst_priorities',
+        'cst_iss_sev_id' => 'cst_severities',
+        'cst_reporter'   => 'cst_reporters',
+        'cst_iss_prc_id' => 'cst_categories',
+        'cst_iss_sta_id' => 'cst_status',
+        'cst_iss_pre_id' => 'cst_releases',
+        'cst_pro_id'     => 'cst_products',
+    ];
+
+    public function up()
+    {
+        $table = $this->table('custom_filter');
+        foreach (self::COLUMNS as $old_name => $new_name) {
+            $table->changeColumn($old_name, 'string', ['length' => 64, 'null' => true]);
```

Comment:
i would use `255` here as there's no special reason to have `64`

Reply:
ACK